client/app/projects/projects.html

```diff
@@ -118,7 +123,7 @@ <h4 class="modal-title" id="addProjectLabel">Add a Project</h4>
         </div>
     </div>
 
-    <div ng-repeat="project in projects">
```

Comment:
this is awesome- if somebody wants to know how to use a filter we should reference this PR